.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,47 @@
+# Publishes a new release of the package
+# whenever there is a new tagged commit to the master branch.
+#
+# The typical workflow for cutting a new release requires executing the following commands:
+# - git checkout master
+# - bumpversion [major/minor/patch] (pick one)
+# - python setup.py sdist bdist_wheel
```

Comment:
Should we mention the step of `rm -rf dist/* build/*` as well?

Reply:
Updated!